libff/algebra/field_utils/field_utils.tcc

```diff
@@ -162,7 +162,7 @@ std::vector<FieldT> pack_bit_vector_into_field_element_vector(const bit_vector &
 template<typename FieldT>
 std::vector<FieldT> pack_bit_vector_into_field_element_vector(const bit_vector &v)
 {
-    return pack_bit_vector_into_field_element_vector<FieldT>(v, FieldT::capacity());
+    return pack_bit_vector_into_field_element_vector<FieldT>(v, FieldT::floor_size_in_bits());
```

Comment:
Why isn't this ceil_size_in_bits?

Reply:
It's because when I updated libff, I only revamped the fields and not the curves, so the curve groups still have the same function names

Reply:
Sorry I misunderstood your comment. It's because the original function used was `capacity()`, which returned `num_bits - 1` for `Fp_model`. This original function was renamed to `floor_size_in_bits()`. If this is an error, it's an error that existed before this change.

Reply:
Lets write an issue for this, I think this is an error.